_episodes/09-cmorization.md

```diff
@@ -617,17 +643,23 @@ If we now run the test recipe on our newly 'CMORized' data,
 esmvaltool run recipe_check_fluxcom.yml --config_file <path to config-user.yml> --log_level debug
 ```
 
-it should be able to find the correct file, but it does not succeed yet. The first
-thing that the ESMValTool CMOR checker brings up is:
+it should be able to find the correct file, but it does not succeed yet. The ESMValTool CMOR checker
+brings up is:
 
 ~~~
-iris.exceptions.UnitConversionError: Cannot convert from unknown units. The
-"units" attribute may be set directly.
+esmvalcore.cmor.check.CMORCheckError: There were errors in variable GPP:
+ GPP: units should be kg m-2 s-1, not unknown
+ lon: standard_name should be longitude, not None
+ lat: standard_name should be latitude, not None
+ lon: units should be degrees_east, not unknown
+ lon: has values < valid_min = 0.0
+ lat: units should be degrees_north, not unknown
+ GPP: does not match coordinate rank
 ~~~
 {: .error}
 
-If you look closely at the error messages, you can see that this error concerns
-the units of the coordinates. ESMValTool tries to fix them automatically,
+If you look closely at the error messages, you can see that these error concern
```

Comment:
suggested change: that these error concern -> the reasons for these errors.

Reply:
@LisaBock  - I am not sure why example_output.txt was touched with this PR as it is not used in the CMORization episode. This is an older version of the file I think. Can you update this from the main repository before pushing?
I have suggseted some changes. If you are ok with them, let me know once when done and I will approve.